src/RouteServiceProvider.php

```diff
@@ -29,6 +29,7 @@ class RouteServiceProvider extends ServiceProvider
      */
     public function boot()
     {
+        require_once __DIR__ . '/Helpers/routes_helpers.php';
```

Comment:
Maybe as part of `3.x`, we could revisit which helpers can be loader earlier in the process (thinking of `register()` instead of `boot()` in `TwillServiceProvider`)

Reply:
Totally. I decided to create helpers because inside a macro we can't reference class methods from the provider, but maybe a static reference would be best?